### DISCUSSION TITLE
How do I compile the solution and run Eto.Test.X?

### DISCUSSION THREAD

**Discussion by eriveraa** [picoe/Eto#2457] [General]
Hi, I have Build all the solution, because i want to test the controls in the Eto.Test.WPF projects but when I try to run the project i got this error:

![image](https://user-images.githubusercontent.com/1510724/229188720-38066bd9-cf7c-4aab-809c-ec21e322e805.png)

**Comment by allsorts46** [comment]
It's likely the other errors below about the missing reference assemblies for .NET Framework 4.62 that are causing the problems. Have you followed the link and downloaded the developer pack as it suggests?
